source/utilities.tex

```diff
@@ -16328,7 +16328,7 @@
 return @\placeholder{INVOKE}@<R>(static_cast<F @\placeholder{inv-quals}@>(f), std::forward<ArgTypes>(args)...);
 \end{codeblock}
 where \tcode{f} is the target object of \tcode{*this} and
-\tcode{f} is lvalue of type \tcode{F}.
+\tcode{F} is the type of that target object.
```

Comment:
Please follow the wording and refer to "f" instead of "that target object" to clarify which "that" we're talking about.

Reply:
fixed